### DISCUSSION TITLE
Redownload files

### DISCUSSION THREAD

**Discussion by Elycoo** [mxpv/podsync#483] [Q&A] [answered]
Hi,
I was increasing the volume of my server and I had to clear some space, so I delete some of the mp3 files of my podcast.

Anyway, now I want to redownload the files again. I was thinking that it would redownload the files automatically but it doesn't. 

How can I trigger podsync to download the missing files?

Elyco.

**Comment by Contextualist** [comment, marked as answer]
Try removing the `db` directory on your config path, then restart podsync. This will reset the download history database.

**Reply by Elycoo** [reply]
Thanks, I was afraid to try this without a recommendation from someone.

**Reply by laurentftech** [reply]
> Try removing the `db` directory on your config path, then restart podsync. This will reset the download history database.

Sorry but where's the config path with docker? I am always wondering how to rebuild the database.

**Reply by Contextualist** [reply]
> where's the config path with docker?

By default the `db` directory is created at the same directory of the config file. If you are using the Docker image following the steps in README, the `db` directory (i.e. `/app/db` inside the container) is not in a volume and hence does not persist. So I guess simply restarting the container throws away the `db` directory.

**Comment by laurentftech** [comment]
Thanks. I may need to do docker rm to throw away the db because with a simple restart it is kept.
